uitestcore/interrogator.py

```diff
@@ -14,16 +14,18 @@ class Interrogator:
     And get_attribute -> String
     """
 
-    def __init__(self, driver, finder, existing_logger=None):
+    def __init__(self, driver, finder, wait_time, existing_logger=None):
```

Comment:
Let's have a default value for this to be consistent with what is in the Waiter class i.e.
wait_time=10